src/internal/kopia/cleanup_backups.go

```diff
@@ -128,11 +128,17 @@ func cleanupOrphanedData(
 		return clues.Wrap(err, "getting all backup models")
 	}
 
-	// assistBackups is the set of backups that have a
-	//   * a label denoting they're an assist backup
-	//   * item data snapshot
-	//   * details snapshot
-	var assistBackups []*backup.Backup
+	var (
+		// assistBackups is the set of backups that have a
+		//   * a label denoting they're an assist backup
+		//   * item data snapshot
+		//   * details snapshot
+		assistBackups []*backup.Backup
+		// mostRecentMergeBase holds the creation time of the most recent merge base
+		// for the given Reason. The Reason -> key translation is done by
```

Comment:
Nit: took me a second to grok the key design from that explanation.
```suggestion
		// mostRecentMergeBase maps the reason to its most recent creation time.
		// The key is created using keysForBackup.
```